secret.tf

```
@@ -0,0 +1,14 @@
+data "azurerm_key_vault" "main" {
+  count = (length(var.frontends) != 0  && var.store_privateip == "true")? 1 : 0
```

can we use a boolean for store_privateip ?

```suggestion
  count = (length(var.frontends) != 0  && var.store_privateip == "true") ? 1 : 0
```